tpgtools/api/cloudbuildv2/samples/gle_priv.connection.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "{{connection}}",
+  "gitlabConfig": {
+    "hostUri": "https://gle-test.proctor-staging-test.com",
+    "serviceDirectoryConfig": {
+      "service": "projects/proctor-gitlab-enterprise/locations/us-west1/namespaces/gle-uw-1/services/private-smoketest"
```

Comment:
Similar to above, hard-coded samples are a bad sign because if these ever go away our tests will fail. Can we set these up in the same test config?